charts/akeyless-api-gateway/values.yaml

```diff
@@ -149,6 +149,52 @@ akeylessUserAuth:
 #  - customer-fragments
 existingSecret: 
 
+
+
+
+# Specifies an initial configuration for the defaults section
+defaultsConf: {}
+  ## Default SAML Access ID to be used for initial WebUI login 
+  # defaultSamlAccessId:
+  ## Default OIDC Access ID to be used for initial WebUI login
+  # defaultOidcAccessId:
+  ## This Default Encryption Key will be selected when creating the following items: Static Secrets, Dynamic Secret Producers and Secret Migration Configurations
+  # defaultEncryptionKey:
+  ## The location of the default path to save secrets
+  # defaultSecretLocation:
+
+
+# Specifies an initial configuration for the caching section
+cachingConf: {}
+    
+  # cacheEnable: true
+  ## The amount of time during which a secret should be cached in minutes
+  # cacheTTL: 60
+
+  # proActiveCacheEnable: false
+  
+  ## The minimum fetching interval in minutes (used to avoid fetching a secret too many times in a given timeframe)
+  # proActiveCacheMinimumFetchingTime: 5
```

Comment:
I would enter all of the proactive chaching values  indented under "proActiveCaching" like this:

proActiveCaching:
  minimumFetchingTime: 5
  dumpInterval: 1

Reply:
done